doc/source/install/build.rst

```diff
@@ -0,0 +1,106 @@
+Build and install
+=================
+
+Once you :doc:`have the source code <get-the-source-code>` and have
+:doc:`activated the virtual environment <virtual-environment>`, you're ready to
+build and install `pycompwa`.
+
+When you install `pycompwa`, you are telling the system where to find it. There
+are two ways of doing this:
+
+(1) by copying the source code and binaries to a folder known to the system
+    (you do this :ref:`with setuptools <setuptools>`)
+(2) by telling the system to directly monitor the :doc:`local repository
+    <get-the-source-code>` as the installation path (we call this
+    :ref:`'developer mode' <install/build:Developer Mode>`).
+
+The second option is more dynamic, because any changes to the source code are
+immediately available at runtime, which allows you to tweak the code and try
+things out. When using the first option, you would have to run ``setuptools``
+again to make the changes known to the system.
+
+
+.. _setuptools:
+
+Using ``setuptools``
+~~~~~~~~~~~~~~~~~~~~
+
+This is easy-peasy! Just navigate to the :doc:`local repository
+<get-the-source-code>` and run:
+
+.. code-block:: shell
+
+  python setup.py install -- -- -j2
+
+where you may change 2 to to the number of cores on your system. The build
+output is written to a folder :file:`_skbuild` and copied to a system folder
```

Comment:
to a system folder -> to the virtual environment

Reply:
https://github.com/ComPWA/pycompwa/pull/83/commits/70b80dde4e4c8340d08ad693bb6023305120fb75

Reply:
Don't know why you dont want to write virtual environment. Because the files are actually being copied there...

Reply:
New formulation is "an installation folder known to the virtual environment". It depends on what one associates with the word environment I guess. For me a virtual environment that's more the act of activating new path variables that point to different install folders, not those install folders themselves.

Reply:
Hmm ok, fair enough. Virtual environment directory would be more simple and precise then

Reply:
https://github.com/ComPWA/pycompwa/pull/83/commits/341e4382529b1cbbda837750082b15465d262b98